engine/src/error.rs

```diff
@@ -9,7 +7,7 @@ use thiserror::Error;
 pub enum EngineError {
     /// Met I/O Error during persisting data
     #[error("I/O Error")]
-    IoError(#[from] io::Error),
+    IoError(String),
```

Comment:
Why changed to `String`? And `I/O Error: {0}`.

Reply:
That's because the error defined in rocksdb is based on `String`. It uses different error message prefixes to tell different errors apart.  Therefore, an `IOError` that throw by rocksdb is only an error with the `Io Error: ...` message.